src/containerapp/azext_containerapp/_cloud_build_utils.py

```diff
@@ -33,9 +33,6 @@ def run_cloud_build(cmd, source, location, resource_group_name, environment_name
     generated_build_name = f"build{run_full_id}"[:12]
     log_in_file(f"Starting the Cloud Build for build of id '{generated_build_name}'\n", logs_file, no_print=True)
 
-    if not os.path.exists(source):
```

Comment:
why remove the check here?

Reply:
This check was moved to _up_utils > _validate_source_artifact_args, so that we can have a logic shared for both --source and --artifact, using Cloud Build or not